src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -77,6 +78,12 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->defaultValue(['%kernel.project_dir%/assets/styles/app.scss'])
                 ->end()
+                ->arrayNode('load_paths')
```

Comment:
To make it possible for people wanting it to use XML config files, you should add `->fixXmlConfig('load_path')` on the parent node (which is the root node)

Reply:
Fixed